packages/gafl-webapp-service/src/services/payment/__test__/govuk-pay-service.spec.js

```diff
@@ -44,7 +44,7 @@ describe('The govuk-pay-service', () => {
     expect(preparePayment({ info: { host: '0.0.0.0:3000' }, headers: { 'x-forwarded-proto': 'https' } }, mockTransaction2)).toEqual({
       amount: 5400,
       delayed_capture: false,
-      description: 'Senior, Salmon and sea trout, 8 days',
+      description: 'Over 65, Salmon and sea trout, 8 days',
```

Comment:
Would be nice to use the SENIOR_MIN_AGE constant from business rules lib for this.

Reply:
It uses the concessions which already use the business rules library that so its circular. This is the for the display of the product it's a to generate the product description throughout. I think bringing in the age here is pointless.

Reply:
I think the benefit is to use the constant in all locations so that, should the concession be relaxed to 60 year olds then the update is only required in one place.